.changeset/wise-wolves-knock.md

```diff
@@ -0,0 +1,5 @@
+---
+'docgen': patch
```

Comment:
Have a feeling this isn't needed. This doesn't get pushed out to NPM or anything, so I think you're good to eliminate this changeset.